src/tests/facilityFeedback.test.js

```diff
@@ -0,0 +1,229 @@
+              const data = {
+                location: 'Test location',
+                address: "Test address",
+                images: 'testImg.jpg',
+                roomNumber: '43',
+                roomDetails: '{Location test details}'
+              };
+              chai
+                .request(app)
+                .post('/facility')
+                .set('authorization', travelAdminToken)
+                .send(data)
+                .end((err, response) => {
+                  facilityId = response.body._facility.id;
+                  expect(response).to.have.status(201);
+                  expect(response.body).to.be.an('object');
+                  done();
+                });
+            });
+         });
+        describe('Facility Feedback APIs', () => {
+          it('It should send a feedback', (done) => {
+            const data = {
+              feedback_title: "Testing the feedback",
+              feedback_content: "This is the dummy feedback content"
+            };
+            chai
+              .request(app)
+              .post(`/facility/feedback/facility/${facilityId}`)
+              .set('authorization', travelAdminToken)
+              .send(data)
+              .end((err, response) => {
+                feedbackId = response.body.feedback.id;
+                expect(response).to.have.status(201);
+                expect(response.body).to.be.an('object');
+                done();
+              });
+          });
+          it('It should fail if one field is missing', (done) => {
+            const data = {
+              feedback_content: "This is the dummy feedback content"
+            };
+            chai
+              .request(app)
+              .post(`/facility/feedback/facility/${facilityId}`)
+              .set('authorization', travelAdminToken)
+              .send(data)
+              .end((err, response) => {
+                expect(response).to.have.status(400);
+                expect(response.body).to.be.an('object');
+                done();
+              });
+          });
+          });
+          describe('GET feedback By Id', () => {
+            it('It should get feedback by Id', (done) => {
+              chai
+                .request(app)
+                .get(`/facility/feedback/${feedbackId}`)
+                .end((err, response) => {
+                  expect(response).to.have.status(200);
+                  expect(response.body).to.be.an('object');
+                  done();
+                });
+            });
+            })
+            describe('GET feedback By facility Id', () => {
+              it('It should get feedback by feedback Id', (done) => {
+                chai
+                  .request(app)
+                  .get(`/facility/feedback/facility/${facilityId}`)
+                  .end((err, response) => {
+                    expect(response).to.have.status(200);
+                    expect(response.body).to.be.an('object');
```

Comment:
Same all through, it is not sufficient to expect the response body to be an object only